modules/site/src/app/examples/button/button-basic-usage.component.spec.ts

```diff
@@ -1,4 +1,11 @@
-import {beforeEach, beforeEachProviders, describe, expect, it, inject,} from '@angular/core/testing';
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  it,
+  inject,
```

Comment:
That's unfortunate.  I just ran the latest `ng format` on the examples app and it did this.  Can we override the formatter styles?  I tend to agree that trailing commas are not helpful.